src/pkg/packager/inspect.go

```diff
@@ -21,6 +21,11 @@ func (p *Packager) Inspect(packageName string, includeSBOM bool, outputSBOM stri
 		return fmt.Errorf("invalid package name: %s", packageName)
 	}
 
+	// If packagePath has partial in the name, we need to combine the partials into a single package
+	if err := p.handleIfPartialPkg(); err != nil {
```

Comment:
Because DeployOpts is not set, this logs `No partial package detected`

And because it doesn't touch package name, results in an error of: `Extract error: format unrecognized by filename: zarf-package-*.tar.zst.part000` from the packager (this error is dogsledded though and should likely be returned to the inspect command for processing as well)